include/erlcloud_aws.hrl

```diff
@@ -7,6 +7,8 @@
           s3_host="s3.amazonaws.com"::string(),
           s3_port=80::non_neg_integer(),
           s3_follow_redirect=false::boolean(),
+          s3_follow_redirect_count=2::non_neg_integer(),
+          s3_bucket_access_method=vhost::vhost|path,
```

Comment:
Why not just replace it? Or is there a reason for keeping the hostname style access?